frontend/src/metabase/archive/components/ArchivedEntityBanner/ArchivedEntityBanner.tsx

```diff
@@ -52,19 +55,26 @@ export const ArchivedEntityBanner = ({
               ).t`This ${entityType} is in the trash.`}
             </Text>
           </Flex>
-          {canWrite && (
+          {hasAction && (
             <Flex gap={{ base: "sm", sm: "md" }}>
               {canRestore && (
                 <BannerButton iconName="revert" onClick={onUnarchive}>
                   {t`Restore`}
                 </BannerButton>
               )}
-              <BannerButton iconName="move" onClick={() => setModal("move")}>
-                {t`Move`}
-              </BannerButton>
-              <BannerButton iconName="trash" onClick={() => setModal("delete")}>
-                {t`Delete permanently`}
-              </BannerButton>
+              {canWrite && (
+                <BannerButton iconName="move" onClick={() => setModal("move")}>
+                  {t`Move`}
```

Comment:
I know this line was not added in this PR, but this could use some `ttag.c` love to clarify that it's a verb not a noun